azavea/summer-of-maps-website
# Post launch updates

### Overview
- Created dedicated yearly project pages (fixes #41)
  - /projects/2017
- Updates copy for header above project shortlist to include the year (fixes #40)
- Added link to all previous project shortlist pages on the projects page
- Project page layout readjusts depending on `_config.yml` fellowship timeframe (fixes #39)
  - Featured projects appear at the top if `student_review` or `fellowship_ongoing` are true
- Added a link to finished projects on project shortlist and shortlist post (fixes #38)
- Homepage hero now has a map visualization (fixes #37)
- Slightly darkened the topographic background (fixes #36)
- Added enhancement to project page to allow setting featured projects (#31) 
- Moved all pages from root directory to `_pages/` directory
- Sets form-modal include to only render when needed
- Updates Readme

#### Demo
**Homepage animated VIZ**
<img width="1259" alt="screen shot 2018-05-03 at 11 52 39 am" src="https://user-images.githubusercontent.com/1928955/39587858-8675ebf8-4ec8-11e8-8811-5fe155041769.png">

**Featured projects/ Project page auto layout adjustment/ updated copy to shortlist on projects page**
![som-projects](https://user-images.githubusercontent.com/1928955/39588166-38fc3cd2-4ec9-11e8-80e0-b9699905dc97.jpg)

**Dedicated project year pages**
<img width="1229" alt="screen shot 2018-05-03 at 11 58 24 am" src="https://user-images.githubusercontent.com/1928955/39588213-592fd57c-4ec9-11e8-952a-94c164316185.png">

**Shortlist final project links**
<img width="1024" alt="screen shot 2018-05-03 at 11 59 28 am" src="https://user-images.githubusercontent.com/1928955/39588287-7f9ee27a-4ec9-11e8-8d0e-b06479f559eb.png">
<img width="1062" alt="screen shot 2018-05-03 at 11 59 33 am" src="https://user-images.githubusercontent.com/1928955/39588288-7faa75ae-4ec9-11e8-883d-a56f2c132700.png">

#### Testing
- Look as the following pages:
  - home
    - check for hero map visualization
  - /projects/
    - check for featured projects and updated layout.
    - You can test the layout feature by changing the current stage to `nonprofit_review` for example
  - /projects/2017
    - these pages are accessible by clicking the headers on the `/projects/all` page
  - /projects/shortlist/2017
    - Check for links to completed projects
